packages/helm-charts/blockscout/templates/blockscout-indexer.deployment.yaml

```diff
@@ -31,6 +31,7 @@ spec:
         - -c
         args:
         - |
+           sleep 5
```

Comment:
Curious (just for my own knowledge) what the need is for this & the other one in the web yaml?

Reply:
My intention was waiting for the migration job to be complete, or at least starting the migration job before any other db process starts, so the db could be dropped if needed. 
Really not very happy with this but cannot think on a simple approach to sync between different pods. Maybe a better approach would be an init container in the indexer that has some capabilities to detect when should the database be dropped and/or migrated.